discord-groups/script.js

```diff
@@ -102,6 +111,24 @@ groupRoles?.addEventListener('click', function (event) {
   }
 });
 
+/**
+ * FOR SEARCHING A GROUP
+ */
+let paragraphElement, paragraphContent;
+const searchInput = document.getElementById('search-groups');
+searchInput.addEventListener('keyup', () => {
+  const searchValue = searchInput.value.toUpperCase();
+  const groupRoles = document.querySelectorAll('.group-role');
+  const groupRolesArray = Array.from(groupRoles);
```

Comment:
need for `Array.from` here? isn't querySelectorAll giving an array

Reply:
Yes, querySelectorAll will give an array. I have removed Array.from